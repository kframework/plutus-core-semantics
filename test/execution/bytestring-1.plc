(program

(module Foo (import) (export () ())
(define b1
  (builtin takeByteString
   5 (builtin dropByteString
     1 (builtin concatenate
       #123456 #789ABC))))

(define b2
  (let
    (lam x (builtin concatenate x #1)) f
      [f #ABC]))
)

)
