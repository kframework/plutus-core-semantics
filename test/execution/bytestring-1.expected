<T>
  <k>
    .
  </k>
  <lenv>
    .Map
  </lenv>
  <genv>
    Foo . b1 |-> #23456
    Foo . b2 |-> #ABC1
  </genv>
  <modules>
    ModuleCellSetItem ( <Module>
      <curr>
        0
      </curr>
      <name>
        Foo
      </name>
      <dataCons>
        .TyExpList
      </dataCons>
      <decl>
        .QualCList
      </decl>
      <importedModules>
        .ModNmList
      </importedModules>
      <exportTy>
        .TyExpList
      </exportTy>
      <exportTm>
        .QualCList
      </exportTm>
    </Module> )
  </modules>
</T>
